test/test_cuda.py

```diff
@@ -200,7 +200,9 @@ def tmp(t):
     ('mode', small_3d, lambda t: [],),
     ('mode', small_3d, lambda t: [1], 'dim'),
     ('remainder', small_3d, lambda t: [3], 'value'),
+    ('remainder', small_3d, lambda t: [-3], 'negative_value'),
     ('remainder', small_3d, lambda t: [small_3d_positive(t)], 'tensor'),
+    ('remainder', small_3d, lambda t: [0 - small_3d_positive(t)], 'negative_tensor'),
```

Comment:
So if you have the `0 - x` it works fine for me as well, which is why I put it like this in the tests, so that they run.
The problem is if I try to use the unary version of the `-` operator, (so just ` - x`) I get the AttributeError that the `neg` function hasn't been implemented.